tests/invariant/Invariant.t.sol

```diff
@@ -71,16 +71,32 @@ contract Invariant_Test is Base_Test, StdInvariant {
             withdrawnAmountsSum += uint256(lockup.getWithdrawnAmount(streamId));
         }
 
+        uint256 aggregateBalance = depositedAmountsSum - refundedAmountsSum - withdrawnAmountsSum;
         assertEq(
             lockup.aggregateBalance(dai),
-            depositedAmountsSum - refundedAmountsSum - withdrawnAmountsSum,
-            unicode"Invariant violation: Σ deposited amounts - Σ refunded amounts - Σ withdrawn amounts != aggregate balance"
+            aggregateBalance,
+            unicode"Invariant violation: aggregate balance != Σ deposits - Σ refunds - Σ withdrawals"
         );
 
         assertGe(
             contractBalance,
-            depositedAmountsSum - refundedAmountsSum - withdrawnAmountsSum,
-            unicode"Invariant violation: contract balances < Σ deposited amounts - Σ refunded amounts - Σ withdrawn amounts"
+            aggregateBalance,
+            unicode"Invariant violation: contract balances < Σ deposits - Σ refunds - Σ withdrawals"
+        );
+    }
+
+    function invariant_DepositedAmountsSumGteAggregateBalance() external view {
```

Comment:
Why do we need this? We already have the following invariant tested above:

`Invariant violation: aggregate balance != Σ deposits - Σ refunds - Σ withdrawals`

This implies `aggregate balance <= Σ deposits`.

Reply:
Good point. We can remove it (feel free to do it yourself).